cli/certinfo/certinfo.go

```diff
@@ -66,8 +71,21 @@ func certinfoMain(args []string, c cli.Config) (err error) {
 		if cert, err = certinfo.ParseCertificateDomain(c.Domain); err != nil {
 			return
 		}
+	} else if c.Serial != "" && c.AKI != "" {
```

Comment:
Seems like we should check `c.DBConfigFile != ""` here as well? Otherwise the user would just see an error about failing to load an empty string?

Reply:
Yeah you're absolutely right! I'll add the same message as when `-db-config` is missing while doing revoke to be consistent.